### PR TITLE
Add docs badge to README

### PR DESCRIPTION
**Pull request by rrrene**
Hi there,

I want to propose to add this badge to the README to show off inline-documentation: [Inline docs](http://inch-ci.org/github/flowjs/flow.js)

The badge links to [Inch CI](http://inch-ci.org) and shows an evaluation by [InchJS](http://trivelop.de/inchjs), a project that tries to raise the visibility of inline-docs. Besides testing and other coverage, documenting your code is often neglected although it is a very engaging part of Open Source.

So far over 500 **Ruby** projects are sporting these badges to raise awareness for the importance of inline-docs and to show potential contributors that they can expect a certain level of code documentation when they dive into your project's code and motivate them to eventually document their own. I would really like to do the same for the **JavaScript** community and roll out support for JS over the coming weeks (early adopters are [forever](https://github.com/foreverjs/forever), [node-sass](https://github.com/sass/node-sass) and [when](https://github.com/cujojs/when)).

Although this is "only" a passion project, I really would like to hear your thoughts, critique and suggestions. Your status page is http://inch-ci.org/github/flowjs/flow.js

What do you think?

### CODE REVIEW

**Comment by AidasK** [comment]
Only 3 suggestions? I think this service managed to detect only 4 functions with a documentation. 
Maybe it could show some more detailed statistics? Maybe function documentation in percent? Maybe it could check if documentation follows jsdoc rules?

**Comment by rrrene** [comment]
Argh, that is very unfortunate. InchJS should detect the functions inside the anonymous function in `src/flow.js` correctly, but it does not. I am embarrassed. :disappointed_relieved:

About the intentions of this: The badge is not really about showing off how good your documentation is or getting to 100%, which is  why there are *no numbers* on it.

The idea is to motivate aspiring Node developers to dive into Open Source projects and read the code. 
It is about *engagement*. And for me, while testing and code coverage are important, inline-docs are the humanly engaging factor in Open Source. This project is about making people less adverse to jumping into the code and see whats happening, because they are not left alone while doing so. I know that, because I put off reading other people's code way too long in my life.

`flow` is seriously good in terms of inline documentation. But even if it wasn't: Most times, any amount of inline-docs is better than none. This badge tries to emphasize that.

**TL;DR** The question for me is: do you like the idea in principle and could I bother you again when the project is ready to parse projects like yours?

**Comment by drzraf** [comment]
Closed by f83b3dde
